dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/managegroups/ManageGroupsCRUDServiceIT.java

```diff
@@ -0,0 +1,122 @@
+package edu.harvard.iq.dataverse.managegroups;
+
+import edu.harvard.iq.dataverse.DataverseServiceBean;
+import edu.harvard.iq.dataverse.DataverseSession;
+import edu.harvard.iq.dataverse.ManageGroupsCRUDService;
+import edu.harvard.iq.dataverse.arquillian.arquillianexamples.WebappArquillianDeployment;
+import edu.harvard.iq.dataverse.authorization.AuthenticationServiceBean;
+import edu.harvard.iq.dataverse.authorization.groups.impl.explicit.ExplicitGroupServiceBean;
+import edu.harvard.iq.dataverse.persistence.dataverse.Dataverse;
+import edu.harvard.iq.dataverse.persistence.group.ExplicitGroup;
+import edu.harvard.iq.dataverse.persistence.user.RoleAssignee;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.transaction.api.annotation.TransactionMode;
+import org.jboss.arquillian.transaction.api.annotation.Transactional;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.ejb.EJB;
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+
+@Transactional(TransactionMode.ROLLBACK)
+@RunWith(Arquillian.class)
+public class ManageGroupsCRUDServiceIT extends WebappArquillianDeployment {
+
+    @PersistenceContext(unitName = "VDCNet-ejbPU")
+    private EntityManager em;
+
+    @EJB
+    private ManageGroupsCRUDService manageGroupsCRUDService;
+    @EJB
+    private DataverseServiceBean dataverseService;
+    @Inject
+    private DataverseSession dataverseSession;
+    @EJB
+    private AuthenticationServiceBean authenticationService;
+    @EJB
+    private ExplicitGroupServiceBean explicitGroupService;
+
+    @Before
+    public void setUp() {
+        dataverseSession.setUser(authenticationService.getAdminUser());
+    }
+
+    @Test
+    public void shouldCreateGroup() {
+        // given
+        Dataverse dv = dataverseService.findByAlias("ownmetadatablocks");
+        RoleAssignee roleAssignee = authenticationService.findAllAuthenticatedUsers().get(0);
+
+        // when
+        manageGroupsCRUDService.create(dv, "testGroup", "testGroupId", "testDesc", Collections.singletonList(roleAssignee));
+
+        // then
+        List<ExplicitGroup> dbExplicitGroupList = explicitGroupService.findByOwner(dv.getId());
```

Comment:
I think that explicitGroupService.findByAlias would be more appropriate. Then you wouldn't have to deal with list.
Then you could do standard asserts of explicitgroup properties by assertEquals

Reply:
I kinda wanted to prove that not only a group was created, but a group for a specific dataverse. Hence getting all groups for a dataverse.

Reply:
Gonna change that after further discussion.

Reply:
If you want to check if this is only group then you can additionally do:
`assertEquals(1, explicitGroupService.findByOwner(dv.getId()).size())`

Reply:
I don't see any changes

Reply:
If you are ok with sticking to `findByOwner` then I'll leave it.

Reply:
I meant: findByAlias + assertions for single group + findByOwner assert size
Now you still have to do tricks with with streams for assertions